cognitedata/reveal
# feat: 360 annotation filter support

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-886

## Description :pencil:


Add capabilities for filtering annotations on status. By default, only `approved` annotations will be loaded in the viewer.

## How has this been tested? :mag:




## Test instructions :information_source:



- Go to https://cog-3d.fusion.cognite.com/3d-test/vision/workflow/review/1388076914295236?env=greenfield and observe that there are 3 annotations, one of which is rejected (temporarily make it approved to see it)
- Go to https://localhost:3000/?project=3d-test&env=greenfield-3d&modelId=4923307033982497&revisionId=3100284276970593
- Load `helideck-site-2-jpeg` 360 image set
- Go to the image besides the lower-most one
- Look down
- Notice that all annotations are visible
- Edit the `annotationFilter.status` field on line 188 in `Image360UI.ts`. Try e.g. `'rejected'`, `'approved'` and `['rejected', 'approved']` as values and observe that the annotations are visible/invisible when expected


## Checklist :ballot_box_with_check:
